spec/imsc-hrm.html

```diff
@@ -556,12 +562,16 @@ <h2>Paint Text</h2>
       <p class='note'>In the case where a property is <em>prohibited</em> in a profile of [[IMSC]], the computed value of the property
       specified in [[ttml2]] can be used.</p>
 
-      <p class='note'>While one-to-one mapping between characters and typographical glyphs is generally the rule in some scripts,
-      e.g. latin script, it is the exception in others. For instance, in arabic script, a character can yield multiple glyphs
-      depending on its position in a word. The Hypothetical Render Model always assumes a one-to-one mapping, but reduces the
-      performance of the glyph buffer for scripts where one-to-one mapping is not the general rule (see GCpy below).</p>
+      <p class='note'>While one-to-one mapping between <a>characters</a> (<a>code points</a>) and glyphs (as defined in [[i18n-glossary]])
```

Comment:
```suggestion
      <p class='note'>While one-to-one mapping between <a>characters</a> and glyphs (as defined in [[i18n-glossary]])
```
_code point_ and _character_ are already linked in the definition of _character_

Reply:
Addressed in https://github.com/w3c/imsc-hrm/commit/ee04753e86c97b5930744c4351e5f440ebd0176e